File: c/99.c
#include <stdio.h>
#include <math.h>

int main(void)
{
	int i, a, b, line;
	double max = 0, prod;
	FILE *input = fopen("houston/input/99", "r");

	for (i = 0; i < 1000; ++i) {
		fscanf(input, "%d", &a);
		fseek(input, 1, SEEK_CUR);
		fscanf(input, "%d", &b);
		fseek(input, 1, SEEK_CUR);

		prod = log(a) * b;
		if (prod > max)
			max = prod, line = i;
	}

	printf("%d\n", ++line);
	return 0;
}

File: js/99.js
var fs = require('fs'),
    input = fs.readFileSync('../input/99'),
    nums = input.toString().split('\n');

var max = 0, line;

for (var i = 0; i < nums.length; ++i) {
    sp = nums[i].split(',');
    a = sp[0] * 1, b = sp[1] * 1;

    prod = Math.log(a) * b;
    if (prod > max)
        max = prod, line = i + 1;
}

console.log(line);
